### PR TITLE
[Bug 15090] Crash when setting acceleratedRendering=true on opencard/openstack

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

**Comment by runrevmark** [comment]
@runrevpanos: Could you chat to @peter-b about this? He fixed this class of bugs in a different way on develop I believe... Basically, any varidiac param lists used in obj-c on 64-bit iOS might be dodgy so it would be better to fix generally if possible, rather than in this specific case.

**Comment by livecodepanos** [comment]
Note: I am preparing a separate pull request against develop-7.0 for this bug, so do not merge this fix into develop-7.0 please

File: engine/src/typedefs.h [hunk 1]
```diff
@@ -182,13 +182,46 @@ typedef uint32_t codepoint_t;
...
+#	endif
+
+#endif
+
+
 #endif
 
 #ifndef nil
-#define nil 0
+
```
Comment (peter-b) [inline engine/src/typedefs.h]: @runrevpanos You can just do:

```
#ifndef nil
#   define nil NULL
#endif
```

**Comment by runrevmark** [comment]
Reviewed-by: @runrevmark.

**Pull request merged**